iothub/device/src/Common/IOThreadScheduler.cs

```diff
@@ -595,9 +595,15 @@ public void DebugVerifyEmpty()
         // by the GC anyway.
         [Fx.Tag.SecurityNote(Critical = "manages NativeOverlapped instance, can be called outside user context")]
         [SecurityCritical]
+
+#if NET451
         unsafe class ScheduledOverlapped
+
+#else
+        class ScheduledOverlapped
```

Comment:
Do we need this class in anything but NET451?
Overlapped operations are always `unsafe`. Can't we just remove the file from ns13/ns20?